lib/lltz_michelson/lltz_michelson.ml

```diff
@@ -87,4 +88,96 @@ and convert_constant (const: LLTZ.E.constant) : Michelson.Ast.t =
     | Bls12_381_g1 _ -> Michelson.T.bls12_381_g1
     | Bls12_381_g2 _ -> Michelson.T.bls12_381_g2
     | Bls12_381_fr _ -> Michelson.T.bls12_381_fr
-    | Signature _ -> Michelson.T.signature
\ No newline at end of file
+    | Signature _ -> Michelson.T.signature
+
+let convert_primitive (prim: LLTZ.P.t) : Michelson.Ast.t =
+  match prim with
```

Comment:
This is an exception to the rule "No (global) openings of modules.": ASTs. Use here an `let open Michelson.Ast.Instruction in`.

Reply:
added ✅